app/(tabs)/toybox/stats/index.tsx

```diff
@@ -5,34 +5,18 @@ import { useAppLanguageContext } from "@/context/AppLanguageContext";
 import { Link } from "expo-router";
 import { useStats } from "@/hooks/useStats";
 import { StyleSheet } from "react-native";
+import { useTable } from 'react-table';
+import React from "react"
 
 function makeTableHTML(myArray: String[][]) {
-  var result = "<table border=1>";
-  result += "<tr><td></td><td>Onset</td><td>Nucleus</td><td>Coda</td></tr>";
```

Comment:
Crashes on mobile will be because of the `<tr>` and `<td>` elements, and also, putting text inside any component other than a `<Text></Text>`

Reply:
They were replaced with React Native tables